gitrevise/merge.py

```diff
@@ -284,7 +302,11 @@ def replay_recorded_resolution(
     ):
         return (b"", None, None)
 
-    (normalized_preimage, conflict_id) = normalize_conflicted_file(preimage)
+    (normalized_preimage, conflict_id) = normalize_conflicted_file(
+        marker_size=marker_size,
```

Comment:
not sure if we are supposed to use kwarg-style calls for non-kwargs?

Reply:
I'm not sure I follow what you mean by non-kwargs?

Reply:
marker_size is a positional argument

Reply:
Sorry, I'm still puzzling over what you mean here. Saying "it's a positional argument" on a line where it's passed by name doesn't make sense. `normalize_conflicted_file` does not have opaque `*args`: there are no positional-only parameters. There's nothing wrong with giving the names explicitly.

Reply:
Oh sorry I mixed that up.
I was just wondering if there is a convention of when to pass the name along an argument.
I think I usually pass the name only for arguments that have default values (and kwargs).
However, we use the name when calling `Repository.bool_config` for example, so this is consistent.